antismash/common/secmet/test/test_locations.py

```diff
@@ -60,9 +60,9 @@ def test_position_conversion_nonzero_compound(self):
         location = CompoundLocation([FeatureLocation(6, 15, strand=-1),
                                      FeatureLocation(21, 27, strand=-1)])
         assert len(location) == 15
-        assert self.func(0, 2, location) == (21, 27)
-        assert self.func(1, 4, location) == (9, 24)
-        assert self.func(3, 5, location) == (6, 12)
+        assert self.func(0, 2, location) == (9, 15)
```

Comment:
That the order of start and end flips is concerning. I would expect that for any given ordered location with a consistent strand, one of `start < end` or `end < start` would apply to all combinations of valid protein locations.

Reply:
again, i agree. if we implement the combined subroutine, this will not be the case anymore. Those tests will be changed to test locations against locations.